stacker/config/__init__.py

```diff
@@ -221,9 +224,11 @@ def validate_parameters(self, data, value):
         if value:
             stack_name = data['name']
             raise ValidationError(
-                    "Stack definition %s contains deprecated "
-                    "'parameters', rather than 'variables'. Please "
-                    "update your config." % stack_name)
+                    "DEPRECATION WARNING: Stack definition %s contains "
```

Comment:
Hmm, technically these were only deprecated in the 0.8 branch. In 1.0, they're not technically deprecated, since they were removed. I think we still have this in here for safety. Is that right @phobologic?

Big 👍 on including a link to the docs though.